enhancements/machine-config/install-time-support-for-image-mode-on-openshift.md

```diff
@@ -0,0 +1,343 @@
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/MCO-1347
+see-also:
+  - "https://github.com/openshift/enhancements/pull/1515/files?short_path=9f0c5f1#diff-9f0c5f1adabad0dfbdb3c9a5b66e53d4fc6619274d7a4c260508d148de17f5c1"
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+
+# Install Time Support for Image Mode on OpenShift
+
+## Summary
+
+This enhancement proposes adding install time support for Image Mode on Openshift, enabling cluster admins to customize the operating system layer during initial cluster installation.
+This feature allows users to build and deploy custom OS images containing additional software, drivers, and configurations from the very beginning of the cluster lifecycle, eliminating the need for post-installation customization and ensuring all nodes start with the desired configuration.
+
+## Motivation
+
+Image Mode on OpenShift became GA in OCP 4.18.21+ and has been well received as a post-install OS customization option. 
+
+We want to bring that support to install time so cluster admins can customize and configure their OS from the beginning instead of having to do it post-install.
+
+OpenShift's current approach to OS management provides excellent consistency and supportability through RHCOS (Red Hat CoreOS), but it requires cluster admins to cede certain configurability aspects to the platform. As workloads become more specialized there is an increasing need for OS-level customization that can be applied from Day 0. An obvious example today is how AI workloads require specific hardware drivers and configurations on nodes.
+
+Some current challenges that Install Time Support For Image Mode on OpenShift would solve:
+- Inability to customize OS images during initial cluster installation
+- Post-installation customization complexity and timing issues
+- Limited support for specialized hardware requirements during installation
+
+### User Stories
+
+#### Story 1: AI/ML Workload Deployment
+
+As a cluster admin deploying AI workloads, I want to include specialized GPU drivers and AI frameworks in my OS image at install time, so that my cluster is ready for AI workloads from the moment it comes online, without requiring post-installation configuration.
+
+#### Story 2: Regulated Environment Deployment
+
+As a cluster admin in a regulated environment, I want to include security agents and compliance tools in my OS image during installation, so that my cluster meets compliance requirements from the moment it comes online, without requiring post-installation configuration.
+
+#### Story 3: Edge Computing Deployment
+
+As a cluster admin deploying edge computing solutions, I want to include edge-specific drivers and monitoring agents in my OS image during installation, so that my edge nodes are immediately functional without requiring network connectivity for post-installation customization.
```

Comment:
Reading the rest of the enhancement, I don't think we're covering for this use case (also applies to the `Limited support for specialized hardware requirements during installation` point above)

This sounds more like custom ISO or built-in registry support that agent team is working on, and we don't explicitly cover this case without additional support outside of the cluster workflow (but perhaps this is also a pre-requisite for that to work end to end?)